frontend/src/store/store.ts

```diff
@@ -1147,6 +1155,13 @@ export default new Vuex.Store<IState>({
       }
     },
 
+    async fetchCharacterVersion({state}, characterId) {
+      const { Characters } = state.contracts();
+      if(!Characters || !state.defaultAccount) return;
+
+      return await Characters.methods.nftVars(characterId, 3).call(defaultCallOptions(state));
```

Comment:
Magic number!

Reply:
Yes it is, but less contract calls. We still have tons of errors on loading and I believe extra calls don't help with that.

Reply:
Same as before